public/app/features/browse-dashboards/components/BrowseActions/DeleteModal.tsx

```diff
@@ -35,6 +35,18 @@ export const DeleteModal = ({ onConfirm, onDismiss, selectedItems, ...props }: P
     <ConfirmModal
       body={
         <>
+          {config.featureToggles.dashboardRestore && (
+            <>
+              <Text element="p">
+                <Trans i18nKey="browse-dashboards.action.delete-modal-restore-dashboards-text">
+                  This action will delete the selected folders immediately but the selected dashboards will be marked
+                  for deletion in 30 days. You can restore the dashboards anytime before the 30 days expires. Folders
+                  cannot be restored.
+                </Trans>
+              </Text>
+              <Space v={2} />
```

Comment:
You'd be able to wrap both `Text` elements into `Stack` and give it a `gap` value instead of using `Space` components.